package.json

```diff
@@ -2,15 +2,16 @@
   "name": "monastic",
   "version": "1.0.1",
   "description": "A functional approach to stateful computations",
-  "main": "index",
+  "type": "commonjs",
```

Comment:
Should [`lint-package-json`][1] assert that `"type"` is specified?


[1]: https://github.com/sanctuary-js/sanctuary-scripts/blob/v3.2.0/bin/lint-package-json